cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -861,12 +930,20 @@ pTxFee =
 pTxBodyFile :: FileDirection -> Parser TxBodyFile
 pTxBodyFile fdir =
   TxBodyFile <$>
-    Opt.strOption
-      (  Opt.long "tx-body-file"
-      <> Opt.metavar "FILE"
-      <> Opt.help (show fdir ++ " filepath of the TxBody.")
-      <> Opt.completer (Opt.bashCompleter "file")
-      )
+    (  Opt.strOption
+         (  Opt.long "out-file"
+         <> Opt.metavar "FILE"
+         <> Opt.help (show fdir ++ " filepath of the TxBody.")
+         <> Opt.completer (Opt.bashCompleter "file")
+         )
+    <|>
+       Opt.strOption
+         (  Opt.long "tx-body-file"
+         <> Opt.internal
+         )
+
+    )
+
```

Comment:
This is problematic as the help text will also denote `out-file` for commands that use `pTxBodyFile` to parse an input file.